.github/actions/compose/docker-compose.smoketest.release.yml

```diff
@@ -32,7 +32,7 @@ services:
       - /var/tmp:/var/tmp
   optimize:
     container_name: optimize
-    image: registry.camunda.cloud/optimize-ee/optimize:${OPTIMIZE_IMAGE_TAG:-latest}
+    image: camunda/optimize:${OPTIMIZE_IMAGE_TAG:-latest}
```

Comment:
Would now already be a good opportunity to stop pushing the `latest` tag for C8? Could already start doing the prefix, as per: https://github.com/camunda/camunda-optimize/issues/13498

cc @OmranAbazid

Reply:
I would keep it separate, I need this PR for the current release